### PR TITLE
feat: add sub composition track

### PR DESCRIPTION
**Pull request by wumaolinmaoan**
## Summary by CodeRabbit

- **New Features**
  - Introduced `SubCompositionClipPlayable` class to enhance sub-composition handling and time updates for `CompositionComponent`.

- **Refactor**
  - Improved handling and initialization of track bindings.
  - Enhanced logic for sub-composition and track data structures in migration processes.
  - Streamlined interface and property assignments for better code maintainability.

### CODE REVIEW

File: packages/effects-core/src/plugins/timeline/tracks/sub-composition-track.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { effectsClass } from 'packages/effects-core/src/decorators';
+import { TrackAsset } from '../track';
+import type { VFXItem } from 'packages/effects-core/src/vfx-item';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/tracks/sub-composition-track.ts:3]: 目录

File: packages/effects-core/src/plugins/timeline/playables/sub-composition-playable-asset.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { effectsClass } from 'packages/effects-core/src/decorators';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/playables/sub-composition-playable-asset.ts:1]: 目录

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/plugins/timeline/tracks/sub-composition-track.ts (hunk 1 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/plugins/timeline/tracks/sub-composition-track.ts]: 已修改

File: packages/effects-core/src/plugins/timeline/playables/sub-composition-clip-playable.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import type { CompositionComponent } from 'packages/effects-core/src/comp-vfx-item';
```
Comment (yiiqii) [inline packages/effects-core/src/plugins/timeline/playables/sub-composition-clip-playable.ts:1]: 目录

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/plugins/timeline/playables/sub-composition-clip-playable.ts (hunk 3 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/plugins/timeline/playables/sub-composition-clip-playable.ts]: 已修改

**Review by yiiqii — APPROVED** [review]

**Pull request merged**